msal/src/Microsoft.Identity.Client/PublicClientApplication.cs

```diff
@@ -418,6 +418,49 @@ public async Task<AuthenticationResult> AcquireTokenAsync(IEnumerable<string> sc
                         behavior, extraQueryParameters, parent, ApiEvent.ApiIds.AcquireTokenWithScopeUserBehaviorAuthority).ConfigureAwait(false);
         }
 
+        /// <summary>
+        /// Non-interactive request to acquire token via Kerberos / Windows Integrated Authentication.
+        /// The username for which the flow is executed is pulled from the operating system, as the current user principal name
+        /// </summary>
+        /// <param name="scopes">Array of scopes requested for resource</param>
+        /// <returns>Authentication result containing token of the current login user</returns>
+        public async Task<AuthenticationResult> AcquireTokenByIntegratedWindowsAuthAsync(IEnumerable<string> scopes)
+        {
+            //TODO: make it platform specific
+            return await AcquireTokenByIWAAsync(scopes, new IWAInput()).ConfigureAwait(false);
+        }
+
+        /// <summary>
+        /// Non-interactive request to acquire token via Windows Integrated Authentication.
```

Comment:
<summary>Non-interactive request to acquire a security token for the signed-in user in Windows, via Integrated Windows Authentication. See https://aka.ms/msal-net-iwa.</summary>
<remark>This overrides enables the application developer to provide a hint about the logged-in user, as on some windows installation the windows administrator might have prevented the applications from accessing the identity of the logged-in user</remark>